platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -124,28 +135,66 @@ public static Result transform(Node nodeXml, TransformerProperties transformerPr
      */
     public static String getRootNamespace(String xml) {
 
-        String rootNamespace = null;
-
         if (xml == null) {
             return null;
         }
 
-        initializeXMLInputFactory();
+        ProcessingResult<String> result = new ProcessingResult();
+        return processElements(xml, result, (xmlStreamReader) -> {
+            result.setValue(xmlStreamReader.getNamespaceURI());
+            result.done();
+        });
+    }
+
+    /**
+     * This method lets clients parse through the elements of an XML document with less
+     * boiler-plate code. This methods handles the configuration of the
```

Comment:
Is the comment that "This method" isn't necessary? I looked at some javadoc examples and I saw they usually start with action like "Returns..." or "Sets..."